packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -258,12 +240,12 @@ export function useGridEditRows(apiRef: GridApiRef) {
       if (!params.isEditable || event.isPropagationStopped()) {
         return;
       }
+      setCellMode(params.id, params.field, 'edit');
```

Comment:
```suggestion

      setCellMode(params.id, params.field, 'edit');
      
```

Just to make it a bit more readable 😉